app/uk/gov/hmrc/platformstatusfrontend/connectors/BackendConnector.scala

```diff
@@ -26,25 +29,30 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class BackendConnector @Inject()(
-  http          : HttpClient,
+  http          : HttpClientV2,
   servicesConfig: ServicesConfig
 )(implicit
    ec: ExecutionContext
-) {
+) extends DefaultBodyWritables:
```

Comment:
Just import the implicits required rather than making the connector a DefaultBodyWritable (which will re-export the implcitis)